vnext/Microsoft.ReactNative/ReactNativeHost.cpp

```diff
@@ -105,6 +107,9 @@ void ReactNativeHost::ReloadInstance() noexcept {
   reactOptions.BundleRootPath = legacySettings.BundleRootPath;
   reactOptions.DeveloperSettings.DebuggerPort = legacySettings.DebuggerPort;
   reactOptions.RedBoxHandler = legacySettings.RedBoxHandler;
+  reactOptions.DeveloperSettings.SourceBundleHost = legacySettings.SourceBundleHost;
+  reactOptions.DeveloperSettings.SourceBundlePort =
```

Comment:
>SourceBundlePort [](start = 33, length = 16)

Could you please change type of this field to uint16_t to avoid the string conversion?

Reply:
Please ignore it. It seems to be a bugger change.